account.go

```diff
@@ -333,15 +333,52 @@ type AccountDeclineSettingsParams struct {
 	CVCFailure *bool `form:"cvc_failure"`
 }
 
-// AccountDocumentsBankAccountOwnershipVerificationParams represents the parameters allowed for passing
-// bank account ownership verification documents on an account.
+// AccountDocumentsBankAccountOwnershipVerificationParams represents the
+// parameters allowed for passing bank account ownership verification documents
+// on an account.
 type AccountDocumentsBankAccountOwnershipVerificationParams struct {
 	Files []*string `form:"files"`
 }
 
+// AccountDocumentsCompanyLicenseParams represents the parameters allowed for
+// passing company license verification documents on an account.
+type AccountDocumentsCompanyLicenseParams struct {
+	Files []*string `form:"files"`
+}
+
+// AccountDocumentsCompanyMemorandumOfAssociationParams represents the
+// parameters allowed for passing company memorandum of association documents
+// on an account.
+type AccountDocumentsCompanyMemorandumOfAssociationParams struct {
+	Files []*string `form:"files"`
+}
+
+// AccountDocumentsCompanyMinisterialDecreeParams represents the parameters
+// allowed for passing company ministerial decree documents on an account.
+type AccountDocumentsCompanyMinisterialDecreeParams struct {
+	Files []*string `form:"files"`
+}
+
+// AccountDocumentsCompanyRegistrationVerificationParams represents the
+// parameters allowed for passing company registration verification documents.
+type AccountDocumentsCompanyRegistrationVerificationParams struct {
+	Files []*string `form:"files"`
+}
+
+// AccountDocumentsCompanyTaxIdVerificationParams represents the parameters
+// allowed for passing company tax id verification documents on an account.
+type AccountDocumentsCompanyTaxIdVerificationParams struct {
```

Comment:
This should use TaxID so `AccountDocumentsCompanyTaxIdVerificationParams` I think (same for resource)